1주차/4796/4096_java_렉스.java

```diff
@@ -0,0 +1,38 @@
+import java.io.*;
+import java.util.*;
+
+public class Main {
+
+    public static void main(String[] args) throws IOException {
+        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
+        StringTokenizer st = new StringTokenizer(br.readLine());
+        StringBuilder sb = new StringBuilder();
+        int caseNum = 1;
+
+        int L = Integer.parseInt(st.nextToken());
+        int P = Integer.parseInt(st.nextToken());
+        int V = Integer.parseInt(st.nextToken());
+
+        while (!isFinal(L, P, V)) {
+            int a = V / P;
+            int b = V % P;
+            if (L < b) {
+                b = L;
+            }
+            int result = a * L + b;
+            sb.append("Case " + caseNum + ": " + result + "\n");
```

Comment:
StringBuilder의 append method를 사용하실거라면 String끼리의 +연산은 사용하지 않는 게 적절하지 않을까요?
```suggestion
            sb.append("Case ")
                .append(caseNum)
                .append(": ")
                .append(result)
                .append("\n");
```

Reply:
오 말해주신대로 수정해보니까 확실히 속도하고 메모리 사용이 향상됐어요!!
감사합니다👍